src/core/public/index.ts

```diff
@@ -102,7 +102,7 @@ export interface CoreStart {
   /** {@link ApplicationStart} */
   application: Pick<ApplicationStart, 'capabilities'>;
   /** {@link ChromeStart} */
-  chrome: ChromeStart;
+  chrome: Pick<ChromeStart, Exclude<keyof ChromeStart, 'getComponent'>>;
```

Comment:
Maybe it's worth having a ChromeStart type and an InternalChromeStart type that extends this. I think that would make the documentation much more readable.